plugins/commands/snapshot/command/root.rb

```diff
@@ -0,0 +1,89 @@
+require 'optparse'
+
+module VagrantPlugins
+  module CommandSnapshot
+    module Command
+      class Root < Vagrant.plugin("2", :command)
+        def self.synopsis
+          "manages snapshots: saving, restoring, etc."
+        end
+
+        def initialize(argv, env)
+          super
+
+          @main_args, @sub_command, @sub_args = split_main_and_subcommand(argv)
+
+          @subcommands = Vagrant::Registry.new
+          @subcommands.register(:save) do
+            require File.expand_path("../save", __FILE__)
```

Comment:
I think we did benchmarks and `require_relative` was faster on Windows than expanding and requiring the absolute path